### PR TITLE
Add enums to markdowngen

### PR DESCRIPTION
**Pull request by hrshdhgd**
Creates .md from enums. Addresses #120

### CODE REVIEW

File: linkml/utils/generator.py [hunk 1]
```diff
@@ -190,6 +193,13 @@ def visit_subset(self, subset: SubsetDefinition) -> None:
         """
         ...
 
+    def visit_enum(self, enum: EnumDefinition) -> None:
```
Comment (cmungall) [inline linkml/utils/generator.py:196]: @hsolbrig - this looks good to me but as this is changing the base class you may want to comment

**Comment by cmungall** [comment]
@hrshdhgd can you show a screenshot or paste in some markdown showing what this looks like? do you need more test cases?

**Comment by hrshdhgd** [comment]
```

# Enum: gp_entity_type_enum




URI: [ontology_association:gp_entity_type_enum](https://w3id.org/ontology_association/gp_entity_type_enum)


## Other properties

|  |  |  |
| --- | --- | --- |

## Permissible Values

protein_coding_gene: 
text: protein_coding_gene 
meaning: SO:0001217 
-------------
ncRNA_gene: 
text: ncRNA_gene 
meaning: SO:0001263 
-------------
mRNA: 
text: mRNA 
meaning: SO:0000234 
-------------
ncRNA: 
text: ncRNA 
meaning: SO:0000655 
-------------
protein: 
text: protein 
meaning: PR:000000001 
-------------
protein-containing complex: 
text: protein-containing complex 
meaning: GO:0032991 
-------------
genetic_marker: 
text: genetic_marker 
meaning: SO:0001645 
-------------

```

**Comment by cmungall** [comment]
would it be possible to render this as a (markdown) table?

**Comment by hrshdhgd** [comment]
You mean like this?

```
| name | text |  meaning |
| protein_coding_gene | protein_coding_gene | SO:0001217  |

**Comment by hrshdhgd** [comment]
The above now looks like this

```
## Permissible Values

| Text | Meaning |
| :--- | --------: |
| protein_coding_gene | SO:0001217 |
| ncRNA_gene | SO:0001263 |
| mRNA | SO:0000234 |
| ncRNA | SO:0000655 |
| protein | PR:000000001 |
| protein-containing complex | GO:0032991 |
| genetic_marker | SO:0001645 |
```

**Pull request merged**